client/src/entrypoints/admin/page-editor.js

```diff
@@ -133,10 +133,10 @@ window.updateFooterSaveWarning = (formDirty, commentsDirty) => {
   }
   clearTimeout(updateFooterTextTimeout);
   const updateWarnings = () => {
-    for (const warning of warnings) {
+    warnings.forEach((warning) => {
       const visible = typeVisibility[warning.dataset.unsavedType];
-      warning.hidden = !visible;
-    }
+      warning.hidden = !visible; /* eslint-disable-line no-param-reassign */
```

Comment:
```suggestion
     // eslint-disable-next-line no-param-reassign
      warning.hidden = !visible;
```

Reply:
Hey, this was my recommendation - I thought it read better with the linting comment inline instead of above in some other suggestions for changes (as it was for params in functions).

In this specific case though I agree with Thibaud here as the comment above should be used and makes sense 99% of the time.